apps/passport-client/new-components/screens/Home/NewHomeScreen.tsx

```diff
@@ -460,7 +469,7 @@ export const NewHomeScreen = (): ReactElement => {
               </_SwipableViews>
               <ButtonsContainer>
                 <SyncIndicator />
-                {Object.keys(appConfig.embeddedZapps).length > 0 && (
+                {getZapps(tickets[currentPos][0]).length > 0 && (
```

Comment:
```suggestion
                {getZapps(tickets[currentPos]?.[0]).length > 0 && (
```

Reply:
on that state we have to have atleast one ticket, if not we render the no events state.